### PR TITLE
Use more concise regexes

### PR DESCRIPTION
**Pull request by underdarknl**
### Changes

Makes the regex simpler and shorter.

### Issue link

https://sonarcloud.io/project/issues?resolved=false&id=minvws_nl-kat-coordination&open=AY_dVgvmGgO8mB8X8BD8

### QA notes

LeakIX normalizer should still make CVE findings with correct finding ID's.

### Code Checklist

- [ ] All the commits in this PR are properly PGP-signed and verified.
- [ ] This PR only contains functionality relevant to the issue.
- [ ] I have written unit tests for the changes or fixes I made.
- [ ] I have checked the documentation and made changes where necessary.
- [ ] I have performed a self-review of my code and refactored it to the best of my abilities.

- [ ] Tickets have been created for newly discovered issues.
- [ ] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [ ] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [ ] I have included comments in the code to elaborate on what is not self-evident from the code itself, including references to issues and discussions online, or implicit behavior of an interface.

---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

**Review by ammar92 — COMMENTED** [review]
Looks good to me. There are also similar issues in:
- octopoes/bits/check_csp_header/check_csp_header.py
- octopoes/bits/spf_discovery/internetnl_spf_parser.py
Perhaps we should include these too?

**Review by ammar92 — DISMISSED** [review]

File: octopoes/bits/check_csp_header/check_csp_header.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def run(resource: HTTPResource, additional_oois: list[HTTPHeader], config: dict[
     # 3: second-level domain
     # 4: end with either a space, a ';', a :port or the end of the string
     #              {1}{ 2}{  3  }{         4       }
-    if re.search(r"\S+\*\.\S{2,3}([\s]+|$|;|:[0-9]+)", csp_header):
+    if re.search(r"\S+\*\.\S{2,3}([\s]+|$|;|:[\d]+)", csp_header):
```
Comment (dekkers) [inline octopoes/bits/check_csp_header/check_csp_header.py:55]:
```suggestion
    if re.search(r"\S+\*\.\S{2,3}([\s]+|$|;|:\d+)", csp_header):
```

File: octopoes/bits/check_csp_header/check_csp_header.py [hunk 2]
```diff
@@ -156,7 +156,7 @@ def _create_kat_finding(header: Reference, kat_id: str, description: str) -> Ite
 def _source_valid(policy: list[str]) -> bool:
     for value in policy:
         if not (
-            re.search(r"\S+\.\S{2,3}([\s]+|$|;|:[0-9]+)", value)
+            re.search(r"\S+\.\S{2,3}([\s]+|$|;|:[\d]+)", value)
```
Comment (dekkers) [inline octopoes/bits/check_csp_header/check_csp_header.py:159]:
```suggestion
            re.search(r"\S+\.\S{2,3}([\s]+|$|;|:\d+)", value)
```

**Comment by stephanie0x00** [comment]
## Checklist for QA:

- [x] I have checked out this branch, and successfully ran a fresh `make reset`.
- [x] I confirmed that there are no unintended functional regressions in this branch:
  - [x] I have managed to pass the onboarding flow
  - [x] Objects and Findings are created properly
  - [x] Tasks are created and completed properly
- [x] I confirmed that the PR's advertised `feature` or `hotfix` works as intended.
- [x] I checked the logs for errors and/or warnings and made issues where necessary

### What works:
Seems to work. Tested with a domain that has various CVEs in LeakIX. The normalizer now runs without issues wrt leaxix. 

### What doesn't work:
n/a

### Bug or feature?:
Other bug identified, but picked up in #3193

**Comment by stephanie0x00** [comment]
## Checklist for QA:

- [x] I have checked out this branch, and successfully ran a fresh `make reset`.
- [x] I confirmed that there are no unintended functional regressions in this branch:
  - [x] I have managed to pass the onboarding flow
  - [x] Objects and Findings are created properly
  - [x] Tasks are created and completed properly
- [x] I confirmed that the PR's advertised `feature` or `hotfix` works as intended.
- [x] I checked the logs for errors and/or warnings and made issues where necessary

### What works:
Data is retrieved from LeakIX and now parsed into CVEs and Findings. 

### What doesn't work:


### Bug or feature?:
Findings for LeakIX do not contain the URL for which it identified the vulnerabilities. This should be added. A new issue is created to fix this bug. See #3211

**Review by dekkers — APPROVED** [review]

**Pull request merged**